.travis.yml

```diff
@@ -4,7 +4,7 @@ language: php
 
 branches:
   except:
-    - /^release-.*$/
+    - /^release-\d+\.\d+\.\d+.*$/
     - /^ghgfk-.*$/
```

Comment:
Just OOI: What's that supposed to do? I can't find a branch or tag with name starting with "ghgfk-"...